README.md

```diff
@@ -28,6 +28,37 @@ Alternatively, add the dependency in your Gemfile:
 gem 'killbill-client', '~> 1.0'
 ```
 
+Authentication
+--------------
+
+A username and password can be set directly in the `options` hash (accepted by each API method):
+```ruby
+options = {
+  :username => 'admin',
+  :password => 'password'
+}
+```
+These credentials are validated by Kill Bill either directly (users managed by Kill Bill) or via a third-party (LDAP, Okta, Auth0, etc.).
+
+Alternatively, a bearer token can be passed as such:
+```ruby
+options = {
+  :bearer => 'token'
+}
+```
+The security token would be validated by Kill bill via a third-party (e.g. Auth0).
+
+By default, Kill Bill won't maintain sessions, except when calling the API `/1.0/kb/security/permissions` (`JSESSIONID`, present in the `Set-Cookie` response header).
```

Comment:
`Won't` or Will ?

Reply:
Will **NOT**. I changed this behavior in 0.20.3 for performance reasons (to avoid creating useless sessions): https://github.com/killbill/killbill/releases/tag/killbill-0.20.3